libs/domain/site/locale-selector/src/locale-selector.def.ts

```diff
@@ -0,0 +1,12 @@
+import { componentDef } from '@spryker-oryx/core';
+
+export const localeSelectorComponent = componentDef({
```

Comment:
maybe we should consider alinging names also?

- localeSelectorComponent
- SiteLocaleSelectorComponent
- localeSelectorSchema